Umbraco-Cloud/Set-Up/Application-Insights/index.md

```diff
@@ -0,0 +1,35 @@
+# Application insigths on Umbraco cloud
+
+## Installing application insights 
+This guide will instruct you how to install Application Insights (AI) for your cloud solution.
+
+### Azure portal
+First thing to have in place when installing AI is ofcourse an AI resource on Azure, so go ahead and open your favorite browser and log onto [Azure Portal](https://portal.azure.com) and spin up a new AI Resource.
+
+After that, you can go ahead and grab your `instrumentation key`, from the AI Resouce overview
+
+![alt text](images/01-Instrumentation-Key.png "Instrumentation Key")
```

Comment:
```suggestion
![alt text](images/01-Instrumentation-key.png "Instrumentation Key")
```
The preview of this image isn't working, think it is due to the image file having a small `k`